core/src/i18n/pl.yaml

```
@@ -128,6 +128,25 @@ settings:
   confirmations: Potwierdzenia
   dontConfirmDelete: Usuwanie bez potwierdzenia
   language: Język
+common:
+  name: Nazwa
+  created: Stworzono
+  labels: Etykiety
+  status: Status
+  buttons:
+    delete: Usuń
+    cancel: Anuluj
+    edit-yaml: Edituj YAML
+  delete-dialog:
+    delete-confirm: Nie pytaj ponownie
+    message: Czy na pewno chcesz usunąć {{type}} {{name}}?
+    title: Usuń {{name}}
+  tooltips:
+    copy-button: Skopiuj do schowka
+    duplicate-key: Zduplikowany klucz
+    k8s-name-input: Nazwa musi zawierać małe znaki alfanumeryczne, może zawierać „-” (np. „my-name1”).
+    key-value: klucz=wartość, musi zaczynać się i kończyć znakiem alfanumerycznym, może zawierać „-”, „_” lub „.”
+    read-file: Odczytuj zawartość pliku jako pojedynczą wartość, z nazwą pliku jako kluczem.
 stateful-sets:
   replicas: Repliki
   title: Zestaw Stanowy
```

If you decide to keep `DaemonSety` (and possibly `ReplicaSety`), then this should also be `StatefulSety`. 
Otherwise, make `Stanowy` lowercase.
```suggestion
  title: Zestaw stanowy
```